docker/yb-voyager-docker

```diff
@@ -152,11 +188,32 @@ do
     i=$(( $i + 1))
 done
 
+# If the metadata file exists then move it to the export_dir and overwrite any existing file.
+if [ -f "$file_name" -a -n "$export_dir" ]
+then
+	mv $file_name $export_dir
```

Comment:
Shall we put it inside export-dir/meta? 
note that you might have to handle the case where the export-dir is empty at this time..